tracer/src/Datadog.Trace.Tools.Runner/Datadog.Trace.Tools.Runner.Standalone.csproj

```diff
@@ -5,7 +5,7 @@
     <Title>Datadog APM Auto-instrumentation Runner</Title>
     <Copyright>Copyright 2020 Datadog, Inc.</Copyright>
     <OutputType>Exe</OutputType>
-    <TargetFrameworks>netcoreapp3.1</TargetFrameworks>
+    <TargetFrameworks>net6.0</TargetFrameworks>
```

Comment:
Any reason we didn't update this to `net5.0` before? 🤔 

AFAIK `net5.0` and `net6.0` use a different single-file behaviour to `netcoreapp3.1`, namely that native files aren't included in the bundle by default (i.e. it's not actually a single file). Should we set `IncludeNativeLibrariesForSelfExtract`? Or even `IncludeAllContentForSelfExtract`?

Also, should we [enable compression for the single file](https://docs.microsoft.com/en-us/dotnet/core/deploying/single-file#compress-assemblies-in-single-file-app)? that has a startup cost, but will reduce file size.

Reply:
hmmm 🤔 that's a good one... why we didn't change it for .NET 5.0???.... I don't recall any reason...

I'll check both properties, thank you. 

The goal for the PR was adding .NET 6.0 build in the tool.
![image](https://user-images.githubusercontent.com/69803/146940517-ed093a79-f1b8-4eb8-9760-b397ef838bf5.png)

But let's solve how we are going to do it for the standalone version as well...

Reply:
I reverted the target framework to `netcoreapp3.1`  for the standalone version, I'll do another PR for this. I prefer to merge this and include .NET 6 to the dd-trace tool first.